news/README.txt

```diff
@@ -37,3 +37,14 @@ You add the following file into the "news" directory, called "9480.new.rst".
     Add a hook for ``foobar``, which has a hidden import.
 
 Done! The changelog entry has been added, and your PR is ready for review.
+
+
+Test Suite
+-----------
+
+To run the test suite, install dependencies in root:
+* pip3 install -e .
+
+Then run the test suite in root:
+* If using Windows, run: `python -m PyInstaller.utils.run_tests`
+* If using Linux, run: `python3 -m PyInstaller.utils.run_tests`
```

Comment:
pip follows the same pattern as python in that it's pip on Windows but pip3 everywhere else. macOS also uses python3 and pip3. I believe the reason is because Linux and mac have both come with python 2.7 preinstalled but that's just a guess.

Reply:
Also don't we need a `pip install -r requirements-test.txt`?

Reply:
@Legorooj  anything else to add?